types/async-redis/index.d.ts

```diff
@@ -0,0 +1,32 @@
+// Type definitions for async-redis 1.0
```

Comment:
```suggestion
// Type definitions for async-redis 1.1
```
https://github.com/moaxaca/async-redis/blob/master/package.json#L3